MerlinAU.sh

```diff
@@ -4610,15 +4610,15 @@ _SelectOfflineZipFile_()
     do
         printf "\nAvailable ZIP files in the directory: [${GRNct}${FW_ZIP_DIR}${NOct}]:\n\n"
 
-        zipFileList="$(ls -A1 "$FW_ZIP_DIR"/*.zip 2>/dev/null)"
+        fileList=="$(ls -A1 "$FW_ZIP_DIR"/*.zip 2>/dev/null)"
```

Comment:
Just found this syntax error that's not noticeable until you look closely (i.e. double equal signs). Submitted PR #284.

Reply:
See, I told you... It's mostly ready 😉 Why you reviewing early!?!? Hahaha LOOL just kidding of course!!

Reply:
> See, I told you... It's mostly ready 😉 Why you reviewing early!?!? Hahaha LOOL just kidding of course!!
>
>
Yeah, I got you!😄

Now, just to clarify so you can understand my perspective.
I wasn't trying to be pedantic when I wrote this before:
>
> OK, let me know when it's fully ready & finalized for review. 
>
In S/W Development & Engineering we have two very specific uses for when code is ready: a) **Ready for review**, b) **Ready to production**.

When making source code commits (AKA pull requests), the code is said to be "**ready for review**" when the author believes it's ready for someone else to double-check the changes/additions for approval, but may not necessarily be ready for production. IOW, there might still be some bugs, or some changes may be required for integration into the main trunk. This is my frame of reference when submitting PRs for review.

Reply:
Oh I don't worry I never thought you were being pedantic. Not with this, or with the user the other day, I always think you just want clarity. 

I agree that the request was unclear yesterday for example, I was just trying to to clarify "my" reply since it seemed like you weren't sure why I would reply what I did.

Here I was just being funny because of course the one PR where I mention "in case you find something" you found something 🤣

Reply:
> Oh I don't worry I never thought you were being pedantic. Not with this, or with the user the other day, I always think you just want clarity.
> 
>
OK, good. Just wanted to make sure we were on the same page WRT the way we communicate. We (s/w devs) have our own specific terminology that we use all the time and is so imprinted into our brains that it becomes 2nd nature.

>
> I agree that the request was unclear yesterday for example, I was just trying to to clarify "my" reply since it seemed like you weren't sure why I would reply what I did.
> 
>
Oh yeah, no worries. I understood that you were clarifying your response based on how you interpreted the request.

> Here I was just being funny because of course the one PR where I mention "in case you find something" you found something 🤣
>
>
Yeah, yeah, I got you!! No problem there, bud. 😜😆

Reply:
I just realized I never mentioned to you but I got s fix for my 6GHz AiMesh, I made a little script to fix it on boot hahaha. I was hoping 388.2_2 would fix it, but considering the hack to fix it is installed on the node and that release just came out, I don't expect another firmware update to fix it anytime soon...

Post in thread 'Asuswrt-Merlin 3006.102.1 is now available for Wifi 7 devices' https://www.snbforums.com/threads/asuswrt-merlin-3006-102-1-is-now-available-for-wifi-7-devices.90788/post-921462

Reply:
> I just realized I never mentioned to you but I got a fix for my 6GHz AiMesh, I made a little script to fix it on boot hahaha. I was hoping 388.2_2 would fix it, but considering the hack to fix it is installed on the node and that release just came out, I don't expect another firmware update to fix it anytime soon...
> 
> Post in thread 'Asuswrt-Merlin 3006.102.1 is now available for Wifi 7 devices' https://www.snbforums.com/threads/asuswrt-merlin-3006-102-1-is-now-available-for-wifi-7-devices.90788/post-921462
>
Nice! Good going bud!!
So it seems that, even after all these years after its introduction, AiMesh is still not reliable enough; it's more like a "work-in-progress" feature. The idea sounds really good & very promising, but ASUS seems to be fumbling with the implementation. And, of course, RMerlin can do nothing about it since it's closed source.

Reply:
> > I just realized I never mentioned to you but I got a fix for my 6GHz AiMesh, I made a little script to fix it on boot hahaha. I was hoping 388.2_2 would fix it, but considering the hack to fix it is installed on the node and that release just came out, I don't expect another firmware update to fix it anytime soon...
> > Post in thread 'Asuswrt-Merlin 3006.102.1 is now available for Wifi 7 devices' https://www.snbforums.com/threads/asuswrt-merlin-3006-102-1-is-now-available-for-wifi-7-devices.90788/post-921462
> 
> Nice! Good going bud!! So it seems that, even after all these years since its introduction, AiMesh is still not reliable enough; it's more like a "work-in-progress" feature. The idea sounds really good & very promising, but ASUS seems to be fumbling with the implementation. And, of course, RMerlin can do nothing about it since it's closed source.

Yes exactly, my hope is this will hold me over until the next release. I figured I'd share the fix since I know many people are impacted. (Basically no one I've talked to has working 6GHz between 102 and 388 AiMesh)

Asus is clearly fumbling the ball considering it's a few nvram values that need to be set, and these are expensive routers .. but we already know that means little, most of their dev power goes to the high selling models, which this is not lol.

While RMerlin can't do anything directly , I hope he can pass on a message to the engineering team (I'm sure they are already aware) but if they know we are going this far to fix it they might bump the priority a bit and put a few more man hours on it